apps/omg_watcher_info/lib/omg_watcher_info/api/transaction.ex

```diff
@@ -112,12 +173,10 @@ defmodule OMG.WatcherInfo.API.Transaction do
   end
 
   defp create_inputs(inputs) do
```

Comment:
Do `create_inputs` and `create_outputs` belong better in the transaction creator module?